scenarios/minicity/scenario.py

```diff
@@ -0,0 +1,66 @@
+import random
+from pathlib import Path
+
+from smarts.sstudio import types as t
+from smarts.sstudio import gen_scenario
+
+NUM_TRAFFIC_FLOWS = 1000
+
+traffic = t.Traffic(
+    flows=[
+        t.Flow(
+            route=t.RandomRoute(),
+            rate=60 * 60,
+            actors={
+                t.TrafficActor(
+                    name="car",
+                    vehicle_type=random.choice(
+                        ["passenger", "bus", "coach", "truck", "trailer"]
+                    ),
+                ): 1
+            },
+        )
+        for _ in range(NUM_TRAFFIC_FLOWS)
+    ]
+)
+
+open_agent_actor = t.SocialAgentActor(
+    name="open-agent", agent_locator="open_agent:open_agent-v0"
+)
+
+laner_actor = t.SocialAgentActor(
+    name="keep-lane-agent", agent_locator="zoo.policies:keep-lane-agent-v0",
+)
+
+travelling_bubbles = [
+    t.Bubble(
+        zone=t.PositionalZone(pos=(50, 0), size=(10, 50)),
+        margin=5,
+        actor=open_agent_actor,
+        follow_actor_id=t.Bubble.to_actor_id(laner_actor, mission_group="all"),
+        follow_offset=(-7, 10),
+    )
+]
+
+static_bubbles = [
+    t.Bubble(zone=t.MapZone((id_, 0, 10), 200, 1), margin=5, actor=laner_actor,)
+    for id_ in [
```

Comment:
NitPick: We may pick some edges on the Bay St. College St or Avenue Road and rename them to make it easier to choose an edge.

Reply:
+1, we should do that. Was going to do later since we'll want to fine-tune this scenario over several PRs.